### PR TITLE
docs(README): explain dash

### PR DESCRIPTION
**Pull request by arnoweiss**
## Description
PRs often fail the pipeline because of the DEPENDENCIES file and the dash licence tool. This adds documentation how to avoid that.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [ ] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,3 +24,20 @@ Specific rules are configured via [.markdownlint.yaml](./.markdownlint.yaml).
...
+  failures in remote services. If that happens, try again tomorrow. If not, continue.
+2. Have java installed. Check by executing `java --version` in your bash terminal. For Windows Powershell or command
+  prompt, this and the following commands may look different but should follow the same structure.
+3. Download the jar of the dash licence tool's latest release (for instance [distributed via eclipse](https://repo.eclipse.org/content/repositories/dash-licenses/org/eclipse/dash/org.eclipse.dash.licenses/)).
+4. Move the file into the top level folder of this repository. Do not add it to git or commit it.
+5. Navigate your terminal to the top level folder of this project (eclipse-tractusx.github.io).
+6. Execute `java -jar ./org.eclipse.dash.licenses-1.1.0.jar package-lock.json -project automotive.tractusx -summary ./DEPENDENCIES`
+  Adjust the version of the jar if necessary. You should see logs saying `[main] INFO Querying X for license data for Y items.`
+  and`[main] INFO Found Z items.`
+7. Commit the changed DEPENDENCIES file and push to your branch again.
```
Comment (mhellmeier) [inline README.md:43]: Use `DEPENDENCIES` instead of DEPENDENCIES.

File: README.md [hunk 2]
```diff
@@ -24,3 +24,20 @@ Specific rules are configured via [.markdownlint.yaml](./.markdownlint.yaml).
 Additionally, there is a npm script `lint-doc`, that will lint all the markdown files inside [docs](./docs) and `lint-kits`, that will lint all the markdown files inside [docs-kits](./docs-kits).
 This script is also run as a pre-commit hook, set up via [husky](https://www.npmjs.com/package/husky).
 You can also run the linting step manually by running `npm run lint-doc` or `npm run lint-kits`.
+
+## Licence Checks
+
+This repository enforces licence compatibility checks on all Pull Requests. Follow these steps if
```
Comment (mhellmeier) [inline README.md:30]: To give the users more content, I would add a link with more information, like TRG 7.04:
https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04#checking-libraries-using-the-eclipse-dash-license-tool

**Review by mhellmeier — APPROVED** [review]
Only some minor suggested adjustments that doesn't block this PR.

File: README.md (hunk 2 above)
Comment (arnoweiss) [reply README.md]: added

File: README.md (hunk 1 above)
Comment (arnoweiss) [reply README.md]: added

**Review by mhellmeier — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -24,3 +24,20 @@ Specific rules are configured via [.markdownlint.yaml](./.markdownlint.yaml).
 Additionally, there is a npm script `lint-doc`, that will lint all the markdown files inside [docs](./docs) and `lint-kits`, that will lint all the markdown files inside [docs-kits](./docs-kits).
 This script is also run as a pre-commit hook, set up via [husky](https://www.npmjs.com/package/husky).
 You can also run the linting step manually by running `npm run lint-doc` or `npm run lint-kits`.
+
+## Licence Checks
```
Comment (FaGru3n) [inline README.md:28]:
```suggestion
## 3rd Party dependeny checks
```

Comment (FaGru3n) [reply README.md]: sorry mist a "c"

File: README.md [hunk 4]
```diff
@@ -24,3 +24,20 @@ Specific rules are configured via [.markdownlint.yaml](./.markdownlint.yaml).
 Additionally, there is a npm script `lint-doc`, that will lint all the markdown files inside [docs](./docs) and `lint-kits`, that will lint all the markdown files inside [docs-kits](./docs-kits).
 This script is also run as a pre-commit hook, set up via [husky](https://www.npmjs.com/package/husky).
 You can also run the linting step manually by running `npm run lint-doc` or `npm run lint-kits`.
+
+## 3rd Party dependeny checks
```
Comment (FaGru3n) [inline README.md:28]:
```suggestion
## 3rd Party dependency checks
```

**Review by FaGru3n — APPROVED** [review]

**Pull request merged**